pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
According to the [package naming guidelines](https://nixos.org/nixpkgs/manual/#sec-package-naming) this package should have
```
version = "2017-02-21";
name = "${pname}-unstable-${version}";
```